packages/mobile/src/screens/signon/ProfileAuto.tsx

```diff
@@ -298,15 +301,18 @@ const ProfileAuto = ({ navigation, route }: ProfileAutoProps) => {
     [twitterInfo, instagramInfo, emailField]
   )
 
-  const signUp = useCallback(() => {
+  const setOAuthInfo = useCallback(() => {
     if (twitterInfo) {
       dispatch(
         signOnActions.setTwitterProfile(
           twitterInfo.twitterId,
           twitterInfo.profile,
           twitterInfo.profile.profile_image_url_https
             ? {
-                uri: twitterInfo.profile.profile_image_url_https,
+                uri: twitterInfo.profile.profile_image_url_https.replace(
+                  /_(normal|bigger|mini)/g,
```

Comment:
OOC why is this needed?

Reply:
Added a comment. It's small by default and they don't expose the hd format one